demo/src/javascripts/components/markdown-page.js

```diff
@@ -4,7 +4,8 @@ import {highlight} from 'highlight.js';
 import React, {Component, PropTypes} from 'react';
 
 const urlRewrites = {
-  '/docs/layers/base-layer.md': '#/layers/base-layer/layer'
+  '/docs/layers/base-layer.md': '#/layers/base-layer/layer',
+  '/docs/custom-layers.md#layerencodepickingcolorindex--number': false
```

Comment:
Again, I think we need some comments on this urlRewrite thing. It is hard to understand what it for someone new.